faunadb/reflect.go

```diff
@@ -23,9 +23,16 @@ func exportedStructFields(aStruct reflect.Value) map[string]reflect.Value {
 			continue
 		}
 
-		fieldName := fieldName(aStructType.Field(i))
+		fieldName, ignore, omitempty, err := parseTag(aStructType.Field(i))
```

Comment:
Is it possible to define `exportedStructFields` using `allStructFields`? They seem to share a lot of code.

Reply:
`allStructFields` contains both `fieldName` and `structTypeField.Name` keys which forces us to implement some processing in `exportedStructFields` func. So it was easier to implement in the current way.

Reply:
Done.